src/types.ts

```diff
@@ -118,6 +118,13 @@ export type Burn = {
   uid: string;
 };
 
+export type DepositDetails = {
```

Comment:
This is what is returned from the endpoint right? If so, it should just be called Deposit. Like Burn, Asset, Transfer

Reply:
Yeah, the whole naming convention around 'deposit' does seem a bit strange to me, as it's actually just returning user details. But will make this change